moe.karaokes.mugen.yml

```diff
@@ -0,0 +1,87 @@
+app-id: moe.karaokes.mugen
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: km-wrapper
+separate-locales: false
+finish-args:
+  - '--filesystem=~/KaraokeMugen:create'
```

Comment:
I'm not a fan of this, and it generally goes against XDG standard. Any reason it doesn't use $XDG_DATA_HOME and $XDG_CONFIG_HOME to store data?

Reply:
Historically speaking we store data created by the app in this folder on *all* systems, since the app is available on Windows, Linux and macOS. I saw in the documentation this wasn't advised, but there are a few reasons this is done like that : 

- The app generates text files for streamers using OBS, who can display information about their ongoing karaoke session by reading those files. They're stored in that folder (among other things).
- Other things are customizable by the user, like player backgrounds, jingles, and so on, so the user needs an easy way to access them.
- It's like an IDE in the sense that users can edit/create their own karaoke files and videos and as such need an easy access to their files when editing them with a third party tool like Aegisub or a video editor of their choice. (to be more precise, the app sotres its data files in that folder, and users can modify them outside of the app as needs be)

We believe `~/KaraokeMugen:create` is about right on how the app works and how to allow users to customize it or work with their data files inside that folder.